client/src/redux/reducers/addSourceWizardReducer.js

```diff
@@ -145,7 +142,7 @@ const addSourceWizardReducer = (state = initialState, action) => {
       return helpers.setStateProp(
         'view',
         {
-          allCredentials: _.get(action, 'payload.data.results', [])
+          allCredentials: (action.payload.data && action.payload.data[apiTypes.API_RESPONSE_CREDENTIALS_RESULTS]) || []
```

Comment:
On the ropes for this one, since its the only use of lodash in the file. You good with me taking a look at the `-es` stuff then in a follow up round, for consistency maybe adding it back in? 

**Side note** - this is one of the files related to work for this upcoming issue/story #1821

Reply:
ok by me